fixtures/load.js

```diff
@@ -0,0 +1,177 @@
+import fs from 'fs';
+import path from 'path';
+import crypto from 'crypto';
+import { MongoClient, ObjectId } from 'mongodb';
+import { promisify } from 'util';
+import { PASSWORD } from '../server/constants/authFactorTypes';
+
+const randomBytes = promisify(crypto.randomBytes);
+const pbkdf2 = promisify(crypto.pbkdf2);
+const readFileAsync = promisify(fs.readFile);
+
+const connectDb = async (uri) => {
+  const client = await MongoClient.connect(uri, { useNewUrlParser: true });
+  return client;
+}
+
+const getAdminRole = async (db) => {
+  const adminRole = db.collection('roles').findOne({ name: 'admin' });
+  if (!adminRole) {
+    throw new Error('Admin Role does not already exist in the database, please make sure you have run all the correct migrations');
+  }
+  return adminRole;
+}
+
+const saveAdminUser = async (db, user) => {
+  const salt = await randomBytes(128);
```

Comment:
Why not use the `generateSalt()` method here from the mongoose `AuthFactor-Password` model? It's a simple static method that would ensure compatibility with future versions.